data/language/en-GB.yml

```diff
@@ -627,7 +627,7 @@ strings:
   606: +{CURRENCY48}
   607: "{CURRENCY48}"
   608: "{COLOUR WINDOW_2}Loan:"
-  609: "{POP16}{POP16}{POP16}{POP16}{CURRENCY32}"
```

Comment:
These strings will have to be adjusted in the other language files as well.

Reply:
Yeah, I wasn't sure on who is doing that or if we have some auto-sync thing going.